R/featEng-entropy.R

```diff
@@ -0,0 +1,58 @@
+#' @title Computing Region Entropy
+#'
+#' @description Computes the entropy of the MRI voxel values for each patient
+#' stratified on individual clusters.
+#'
+#' @details \code{computeRegionEntropy()} takes in two arguments: a \code{tibble}
+#' voxel_df and a \code{tibble} data_df. \code{voxel_df} is assumed to be in long
+#' format, with each individual pixel being represented on a single row, so that
+#' the first columns is 'pid,' columns 2 to 4 are 'x', 'y', 'z', and column 5
```

Comment:
We should not have the user order columns this way. The code should run as long as the column names are correctly specified, not the names AND the order. The user should do as little work as possible